chaoscenter/web/config/webpack.dev.js

```diff
@@ -38,29 +38,29 @@ const devConfig = {
       }
     },
     proxy: {
-      '/api': {
-        pathRewrite: { '^/api': '' },
-        target: process.env.CHAOS_MANAGER
-          ? process.env.CHAOS_MANAGER
-          : targetLocalHost
-          ? 'http://localhost:8080'
-          : `${baseUrl}/api`,
-        secure: false,
-        changeOrigin: true,
-        logLevel: 'info'
-      },
-      '/auth': {
-        pathRewrite: { '^/auth': '' },
-        target: process.env.CHAOS_MANAGER
-          ? process.env.CHAOS_MANAGER
-          : targetLocalHost
-          ? 'http://localhost:3000'
-          : `${baseUrl}/auth`,
-        secure: false,
-        changeOrigin: true,
-        logLevel: 'info'
-      }
-    }
+  '/api': {
+    pathRewrite: { '^/api': '' },
+    target: process.env.CHAOS_MANAGER
+      ? process.env.CHAOS_MANAGER
+      : targetLocalHost
+      ? 'http://localhost:8080'
+      : `${baseUrl}/api`,
+    secure: false,
+    changeOrigin: true,
+    logLevel: 'info'
+  },
+  '/auth': {
+    pathRewrite: { '^/auth': '' },
+    target: process.env.CHAOS_MANAGER
+      ? process.env.CHAOS_MANAGER
+      : targetLocalHost
+      ? 'http://localhost:3000'
+      : `${baseUrl}/auth`,
+    secure: false,
+    changeOrigin: true,
+    logLevel: 'info'
+  }
+}
```

Comment:
This is just formatting change. Can you update the prettiers with the UI configuration?